br.gov.cti.invesalius.json

```diff
@@ -0,0 +1,203 @@
+{
+    "app-id": "br.gov.cti.invesalius",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
```

Comment:
Update runtime to 3.30

Reply:
Looking at the dependencie, it seems like `org.freedesktop.Platform` should be enough.

Reply:
I'm using "org.gnome.Platform" because WXPython needs Webkitgtk, I wasn't able to compile wxpython without webkitgtk. About using the runtime version 3.30 it was because I was not able to compile Scipy with this version and gfortran.

Reply:
> About using the runtime version 3.30 it was because I was not able to compile Scipy with this version and gfortran.

 Error message? (I might have an idea why it failed)

Reply:
@eszlari this the error message: error: Requested extension org.freedesktop.Sdk.Extension.gfortran-62 not installed

Reply:
if I use the 3.28 runtime version this problem doesn't happen

Reply:
That's because gfortran is part of 18.08 Sdk/Platform (and anything based on it like GNOME 3.30 runtime), you don't need extension for it.